### ISSUE TITLE
BigValue Format Tag Error on data update

### ISSUE THREAD

**Issue by mcrascal** [evidence-dev/evidence#1018]
The most basic way to reproduce:

```
<script>
  import { BigValue } from '@evidence-dev/core-components';

  let testData = [{ test: 1 }];
  setTimeout(() => {
    testData = [{ test: 2 }];
  }, 1000);
</script>

<BigValue data={testData} value="test" />

```

on the second render (after the 1000ms timeout), I get an exception:

`Uncaught (in promise) potentialFormatTag.toLowerCase is not a function`

**Comment by archiewood** [comment]
I've just repro'd this error again:

https://github.com/evidence-dev/northstar-report


1. Run the project
1. Change some sql

Error:


![CleanShot 2023-07-19 at 17 15 37](https://github.com/evidence-dev/evidence/assets/58074498/f302692c-6609-446c-8e9a-aea033460fe9)

**Comment by csjh** [comment]
@archiewood I don't think the fix has released yet, still in #1045

**Comment by archiewood** [comment]
Ah, that tracks
